src/main/java/Baymax.java

```diff
@@ -0,0 +1,163 @@
+import java.util.ArrayList;
+import java.util.PrimitiveIterator;
+import java.util.Scanner;
+import task.*;
+public class Baymax {
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+    public static void main(String[] args) {
+        //instantiate Baymax
+        Baymax baymax = new Baymax();
+
+        //Scanner object for user input
+        Scanner scanner = new Scanner(System.in);
+        String userInput;
+
+        //load Tasks from file
+        taskList = Storage.loadTasks();
+
+        //Display greeting
+        String line = "______________________________________________________________\n";
+        System.out.println(line
+                + "Hello! I'm Baymax, your personal emotional companion :)\n"
+                + "How may I help you today?\n"
+                + line);
+
+        //Input loop
+        while (true) {
+            userInput = scanner.nextLine();
+
+            //Check for end of conversation
+            if (userInput.equalsIgnoreCase("bye")) {
+                //Display exit message
+                System.out.println(line
+                        + "Goodbye. Have a nice day, and remember;\n"
+                        + "don't forget to smell the flowers, as you aim for the sky\n"
+                        + line);
+                break;
+            }
+            //Check for list
+            if (userInput.equalsIgnoreCase("list")) {
+                baymax.printList();
+                continue;
+            }
+            //mark task
+            if (userInput.startsWith("mark")) {
+                int taskNumber = Integer.parseInt(userInput.split(" ")[1]);
+                baymax.markAsDone(taskNumber);
+                continue;
+            }
+            //unmark task
+            if (userInput.startsWith("unmark")) {
+                int taskNumber = Integer.parseInt(userInput.split(" ")[1]);
+                baymax.unmark(taskNumber);
+                continue;
+            }
+            //add Todo
+            if (userInput.startsWith("todo ")) {
+                baymax.addTodo(userInput.substring(5));
+                continue;
+            }
+            //add Deadline
+            if (userInput.startsWith("deadline ")) {
+                String[] parts = userInput.substring(9).split(" /by ");
+                baymax.addDeadline(parts[0], parts[1]);
+                continue;
+            }
+            //add Event
+            if (userInput.startsWith("event ")) {
```

Comment:
Is there a reason why theses 3 commands have whitespace after them? You can consider stripping the string prior.